### ISSUE TITLE
Unselecting a row in DetailsList is not announced when using shift + up key on a selected row

### ISSUE THREAD

**Issue by sushruth** [microsoft/fluentui#12296]
### Environment Information

- **Package version(s)**: 7.77.0
- **Browser and OS versions**: Edge Win10

### Describe the issue:

While going down a list of items (with shift + down arrow) in a selectable DetailsList, selecting a row is annouced but unselecting (shift + up arrow) is not announced

#### Actual behavior:

Unselecting a row isnt being announced

#### Expected behavior:

Unselecting a row should be announced

### If applicable, please provide a codepen repro:

The main usage example in uifabric docunmentation serves as a repro - https://uifabric.azurewebsites.net/#/controls/web/detailslist

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? NO

Requested priority: Normal

Products/sites affected: NA

**Comment by aneeshack4** [comment]
@sushruth Thanks for reaching out to us - you should be using CTRL + ALT + ARROWs to be navigating DetailsList.

**Comment by sushruth** [comment]
@aneeshack4 Thanks. Is there a document or standard spec that you refer to while verifying DetailsList's accessibility? Can you please share that?

**Comment by aneeshack4** [comment]
@sushruth These are keyboard shortcuts that are standard across the web, Narrator & other screenreaders/ATs probably have them in their documentation/walk throughs. I'm not aware of any others but I've asked around and will let you know if I hear anything.

**Comment by aneeshack4** [comment]
I haven't heard anything else @sushruth - I'd suggest going to the ATs' documentation for these keyboard shortcuts.
